packages/lib-sourcify/test/compiler/vyperCompiler.ts

```diff
@@ -10,7 +10,10 @@ import { logDebug, logError, logWarn } from '../../src/lib/logger';
 const HOST_VYPER_REPO = 'https://github.com/vyperlang/vyper/releases/download/';
 
 export function findVyperPlatform(): string | false {
-  if (process.platform === 'darwin') {
+  if (
+    process.platform === 'darwin' &&
+    (process.arch === 'x64' || process.arch === 'arm64')
+  ) {
     return 'darwin';
   }
   if (process.platform === 'linux' && process.arch === 'x64') {
```

Comment:
Did you check anything for windows?

Reply:
Yes, I also checked Windows. I ran the `file` command to check for which architectures the three binary types are.